docs/configuration.md

```diff
@@ -1,6 +1,6 @@
 # Configuration
 
-Cloudbroker config is added in the default portal config files located at `/opt/cfg/portals/main/config.yaml`
+Cloudbroker config is added in the default portal config files located at `/optvar/cfg/portals/main/config.yaml`
```

Comment:
Perhaps make it <cfgdir> instead? Unfortunately, /optvar/cfg is not always the case of config

Reply:
j.dirs.CFGDIR :D